trunk/src/core/srs_core_deprecated.hpp

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright (c) 2013-2024 The SRS Authors
+//
+// SPDX-License-Identifier: MIT
+//
+
+#ifndef SRS_CORE_DEPRECATED_HPP
+#define SRS_CORE_DEPRECATED_HPP
+
+#include <srs_core.hpp>
```

Comment:
this include header,`srs_core.hpp`, is useless, can be removed.

Reply:
I want to make sure the first included file is this one.